### PR TITLE
Bump and changelog for 3.2. release.

### PR DESCRIPTION
**Pull request by troessner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 3.2.0 (2015-08-17)
+
+* (mvz) Also detect attribute writers created with attr
+* (beanieboi) Report attr_writer and attr_accessor in Attribute smell
```
Comment (mvz) [inline CHANGELOG.md]: Maybe switch these around so they make sense together?

Comment (troessner) [inline CHANGELOG.md]: Yes, very dusselig from me, fixed.

**Comment by troessner** [comment]
Addressed all comments - > merge and release?

**Comment by chastell** [comment]
I restarted the build in which our dear friend Rubinius decided to barf some internal error.

**Comment by chastell** [comment]
Oh now it blows up on the `reek inline.rb optparse.rb redcloth.rb` example with a SIGTERM (parsing timeout?). Restarting again…

**Comment by chastell** [comment]
Oh how I love testing by force – rerunning tests until green. Merging!

**Comment by troessner** [comment]
Released 3.2.!

**Pull request merged**